src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -0,0 +1,495 @@
+        \end{pmatrix}
+
+    The ODE system has no parameters.
+    This implementation includes the Jacobian :math:`J_f` of :math:`f`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(3, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`3`-dimensional mean vector and
+        :math:`3 \times 3`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    Returns
+    -------
+    IVP
+        IVP object describing the rigid body dynamics IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return rigidbody_rhs(t, y)
+
+    def jac(t, y):
+        return rigidbody_jac(t, y)
+
+    return IVP(timespan, initrv, rhs, jac=jac)
+
+
+def rigidbody_rhs(t, y):
+    y1, y2, y3 = y
+    return np.array([y2 * y3, -y1 * y3, -0.51 * y1 * y2])
+
+
+def rigidbody_jac(t, y):
+    y1, y2, y3 = y
+    return np.array([[0.0, y3, y2], [-y3, 0.0, -y1], [-0.51 * y2, -0.51 * y1, 0.0]])
+
+
+def vanderpol(timespan, initrv, params=0.1):
+    r"""Initial value problem (IVP) based on the Van der Pol Oscillator.
+
+    The Van der Pol Oscillator is defined through
+
+    .. math::
+
+        f(t, y) =
+        \begin{pmatrix}
+            y_2 \\
+            \mu \cdot (1 - y_1^2)y_2 - y_1
+        \end{pmatrix}
+
+    for a constant parameter  :math:`\mu`.
+    :math:`\mu` determines the stiffness of the problem, where
+    the larger :math:`\mu` is chosen, the more stiff the problem becomes.
+    Default is :math:`\mu = 0.1`.
+    This implementation includes the Jacobian :math:`J_f` of :math:`f`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(2, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`2`-dimensional mean vector and
+        :math:`2 \times 2`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float), optional
+        Parameter :math:`\mu` for the Van der Pol Equations
+        Default is :math:`\mu=0.1`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the Van der Pol Oscillator IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return vanderpol_rhs(t, y, params)
+
+    def jac(t, y):
+        return vanderpol_jac(t, y, params)
+
+    return IVP(timespan, initrv, rhs, jac=jac)
+
+
+def vanderpol_rhs(t, y, params):
+    y1, y2 = y
+    if isinstance(params, float):
+        mu = params
+    else:
+        (mu,) = params
+
+    return np.array([y2, mu * (1.0 - y1 ** 2) * y2 - y1])
```

Comment:
one thing I am always confused about with van der pol is whether it is mu (1 - y1^2)y1 - y2 or whether it is mu((1-y1^2)y1 - y2). I know the sources suggest the former, but https://benchmarks.sciml.ai/html/StiffODE/VanDerPol.html suggests the latter. In my experiments, I only got the stiff versions to work with the latter. @nathanaelbosch do you know anything about this? You are the van der pol expert.

Reply:
Phew apparently I never caught that :/ I did use the latter in my experiments, but I don't have any tests about the former. Thanks a lot for pointing that out!